src/plugins/workspace/public/components/utils/workspace.ts

```diff
@@ -23,6 +23,12 @@ export const navigateToWorkspaceDetail = (
   );
 };
 
+export const navigateToWorkspaceListWithUseCase = ({ application }: Core, useCaseTitle: string) => {
+  const url = application.getUrlForApp(WORKSPACE_LIST_APP_ID, { absolute: true });
+  const search = `/?useCase=${useCaseTitle}`;
```

Comment:
can we use hash, so that router change is not needed? similar with workspace detail page

Reply:
Updated. This may just be a boilerplate for now, it actually depends on how @yubonluo uses later.

Reply:
If necessary in the future, I will make some adjustments when integrating it to the homepage